src/NHSD.BuyingCatalogue.Identity.Api/Models/ApplicationUser.cs

```diff
@@ -5,14 +5,107 @@ namespace NHSD.BuyingCatalogue.Identity.Api.Models
 {
     public sealed class ApplicationUser : IdentityUser
     {
-        public string FirstName { get; set; }
-
-        public string LastName { get; set; }
+        public string FirstName { get; }
 
+        public string LastName { get; }
+        
         public string DisplayName => $"{FirstName} {LastName}";
 
-        public Guid PrimaryOrganisationId { get; set; }
+        public Guid PrimaryOrganisationId { get; }
+        
+        public OrganisationFunction OrganisationFunction { get; }
+
+        public bool Disabled { get; private set; }
+
+        public bool CatalogueAgreementSigned { get; private set; }
+        
+        private ApplicationUser()
+        {
+        }
```

Comment:
I thought it was needed for Entity Framework, otherwise how does it create the object with a private constructor containing parameters? 

I took it out, ran the tests again, no problems 👍

Reply:
I wonder if you were right and this is required given the failed checks although that seems odd given that the tests passed when you ran them...